test/features/conditionalLogic.feature

```diff
@@ -12,7 +12,10 @@ Feature: Set of scenarios testing show & hide functionality and conditional logi
   Scenario: field visible when show & hide condition is met
     Given a case type containing conditionals exists
     When I enter 'showmethemoney' into the 'text' field
-    Then the field with label 'Text Field 2' is visible
+    Then the field with label 'Text Field 2' is visible with grey bar
+    And I click the Continue button
+    And the field with label 'Text Field 3' is visible
+    And the field with label 'Text Field 3' is visible without grey bar
```

Comment:
Why two similarly looking steps? Wouldn't it be enough to have the last one only?

Reply:
@andrewfolga I think that was done because we wanted to have two asserts, one for the text field visible and there other for field not having a grey bar. We could combine those two but I thought keeping it separate will probably help min reusing the methods somewhere else. We will review this again later.

Reply:
In that case should not the last step sound more like:
`And the field with label 'Text Field 3' is without grey bar`
just omitting the visible?

Reply:
Also it is breaking the not more then 5 steps per scenario rule.

Reply:
Done.